src/Audit/AuditOSSIndex.ts

```diff
@@ -92,6 +102,44 @@ export class AuditOSSIndex {
     result.vulnerabilities && printVuln(result.vulnerabilities);
   }
 
+  private filterVulnerabilities(results: Array<OssIndexServerResult>): Array<OssIndexServerResult> {
```

Comment:
This could either be lifted to some new class, I think, which would make it a bit easier to test. Right now the concerns are a bit mixed, maybe we should lift it out?

Reply:
I did this in f167dd6